modules/game/src/main/JsonView.scala

```diff
@@ -72,7 +72,10 @@ object JsonView {
   }
 
   implicit val crazyhouseDataWriter: OWrites[chess.variant.Crazyhouse.Data] = OWrites { v =>
-    Json.obj("pockets" -> List(v.pockets.white, v.pockets.black))
+    Json.obj("pockets" -> Json.obj(
+      "white" -> v.pockets.white,
+      "black" -> v.pockets.black
+    ))
```

Comment:
I think these server-side changes might break compability with the mobile app. Nice simplifications otherwise.

Reply:
I'll leave these writers how they were. No reason to risk breaking compatibility